src/components/YearSelector.js

```diff
@@ -52,7 +53,12 @@ class YearSelector extends Component {
 
     return (
       <div>
-        <p> Select Year of Data to be displayed: </p>
+        {window.innerWidth > mobileStyle ? (
+          <p>Select Year of Data to be displayed:</p>
+        ) : (
+          {}
```

Comment:
This empty block of code should be removed thou, that's what i meant from the previous comment